Makefile

```diff
@@ -72,7 +76,17 @@ drush:
 	$(call message,Executing \"drush -r /var/www/html/web $(COMMAND_ARGS) --yes\")
 	$(call docker-www-data, php drush -r /var/www/html/web $(COMMAND_ARGS) --yes)
 
-prepare: | up
+configuring_env:
+    # If OS Linux - change PHP_TAG in environment.
+	@if [ $(OS_NAME) = "Linux" ]; then \
+		echo "${cyan}${bold}$(PROJECT_NAME): Defined OS LINUX. Change PHP_TAG for docker image in environment${reset}" \
+		# Uncomment all the strings containing 'PHP_TAG'. \
+		sed -i '/PHP_TAG/s/^# //g' \.env; \
+        # Comment all the strings containing 'PHP_TAG' and '-dev-macos-. \
+        sed -i -E '/PHP_TAG.+-dev-macos-/s/^/# /g' \.env; \
```

Comment:
Smth is wrong with indentations here (indents are not the same for the lines in one block) - check where you use tabs and where the blank symbols. Tabs are essential for Makefile syntax (first tab in a line tells make that it is an instruction to execute / shell command), but once you started the shell command and use the multiline syntax with '\' at the end of lines - then all the indents of the next line are going into this shell command.